sqld/src/query_analysis.rs

```diff
@@ -35,6 +35,19 @@ fn is_temp(name: &QualifiedName) -> bool {
     name.db_name.as_ref().map(|n| n.0.as_str()) == Some("TEMP")
 }
 
+fn is_reserved_tbl(name: &QualifiedName) -> bool {
+    let n = &name.name.0;
+    n == "_litestream_seq" || n == "_litestream_lock" || n == "libsql_wasm_func_table"
+}
+
+fn write_if_not_reserved(name: &QualifiedName) -> Option<StmtKind> {
+    if is_reserved_tbl(name) {
+        None
+    } else {
+        Some(StmtKind::Write)
+    }
```

Comment:
I also can't resist posting that this whole function body can be replaced with the following idiomatic beauty:
```rust
fn write_if_not_reserved(name: &QualifiedName) -> Option<StmtKind> {
    is_reserved_tbl(name).then(|| StmtKind::Write)
}
```
(obviously non-blocking though)